Kentor.AuthServices.HttpModule/HttpRequestBaseExtensions.cs

```diff
@@ -5,6 +5,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.Web;
+using Kentor.AuthServices.Configuration;
```

Comment:
Minor: should be no change to this file now that you've taken the different approach.